runtime/doc/options.txt

```diff
@@ -3267,7 +3267,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 				     >:SignColumn,B:SpellBad,P:SpellCap,
 				     R:SpellRare,L:SpellLocal,-:Conceal,
 				     +:Pmenu,=:PmenuSel,x:PmenuSbar,
-				     X:PmenuThumb")
+				     X:PmenuThumb,q:QuickFixLine")
```

Comment:
> I actually think that the output of snapshot_util() should be what's displayed for failed test, but followed with a list of the highlights instead of them being inline with the screen text.

But that is exactly the output of `snapshot_util()` _today_. There is no need whatsoever to "focus on" or "care about" relevant or irrelevant highlights (in the part of generating the actual expects, I mean), just copy the entire list of highlights to the set_default.